giskard_vision/core/dataloaders/hf.py

```diff
@@ -101,3 +111,37 @@ def cleanup(self):
         Clean the temporary folder
         """
         shutil.rmtree(self.temp_folder)
+
+    @abstractmethod
+    def get_raw_hf_image(self, idx: int) -> PILImage:
```

Comment:
are images always in the PIL type from HF?

Reply:
Yes, it decodes any formats and return a PIL image to use in HF pipeline